### PR TITLE
Updates to STIS calibration wrapper and plots

### PR DESCRIPTION
**Pull request by jotaylor**
Improvements were made to the STIS calibration wrapper (not to the calibration code itself) and the plot generation script.

### CODE REVIEW

File: ullyses/calibrate_stis_data.py [hunk 1]
```diff
@@ -385,27 +390,30 @@ def make_plots(self):
...
         if self.x1d_mast is not None:
             self.plots_made = True
-            print("\n", f" CREATING DIAGNOSTIC PLOTS ".center(NCOLS, SYM), "\n")
             mastx1d = self.x1d_mast
             for target, target_pars in self.target_dict.items():
                 customx1d = target_pars['out_x1d']
                 pdffile = plot_stis_data.plot_all_x1d(customx1d, mastx1d, target, self.outdir)
                 self.target_dict[target]["oned_plot"] = pdffile
+        else:
+            self.plots_made = True
```
Comment (stscirij) [inline ullyses/calibrate_stis_data.py:403]: Or this one?

**Review by stscirij — DISMISSED** [review]
From what I can tell, this looks fine

File: ullyses/calibrate_stis_data.py [hunk 2]
```diff
@@ -385,27 +390,30 @@ def make_plots(self):
         Plot calibration output.
         """
 
+        print("\n", f" CREATING DIAGNOSTIC PLOTS ".center(NCOLS, SYM), "\n")
+        self.plots_made = True
         if self.x1d_mast is not None:
             self.plots_made = True
```
Comment (stscirij) [inline ullyses/calibrate_stis_data.py:396]: Don't need this line any more

Comment (jotaylor) [reply ullyses/calibrate_stis_data.py:396]: Good catch! I made the first declaration of `self.plots_made` to be False as a workaround. Even though it's always set to True for now, there may be a future case when I need to leave it as False, so I left the check in for now.

**Comment by jotaylor** [comment]
@stscirij can you re-review after my latest update?

**Review by Rplesha — APPROVED** [review]
Seems reasonable to me

**Pull request merged**